README.md

```diff
@@ -384,5 +385,17 @@ If the result of applying the template does not match with the block sample in H
 
 All tests will be run automatically using [Travis](https://travis-ci.org) for each pull request.
 
+<a name="workflow"></a>
+### Workflow
+
+The issue list included in the current scope is displayed on a special [Agile Board](waffle.io/bem/bem-components).
+
+Issues on the Agile Board can be in four status:
+
+- backlog — issues that need to disassemble, evaluate and make a decision about their implementation. In this status can also be issues that need clarifying information.
+- ready — disassembled issues with a specific assignee, the decision about which implementation is adopted.
```

Comment:
«ready — approved issues»

BTW, haven't we agreed to rename «ready» with «approved»?